config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins 'https://beta-yacht-rental.netlify.app' # later change to the domain of the frontend app
```

Comment:
- [ ] The local dev won't work because CORS will block it.

This is because the only origin that is being configured is the live demo frontend.  Kindly, either reflect this on your instructions in case your users want to run local dev, or add the local origin that is missing. 👍